src/univariates.jl

```diff
@@ -376,36 +395,41 @@ function cdf(d::DiscreteUnivariateDistribution, x::Integer, ::Type{FiniteSupport
     return c
 end
 
-cdf(d::DiscreteUnivariateDistribution, x::Real) = cdf(d, floor(Int,x))
-cdf(d::ContinuousUnivariateDistribution, x::Real) = throw(MethodError(cdf, (d, x)))
-
+cdf(d::UnivariateDistribution{<:ContiguousSupport}, x::Real) =
+    cdf(d, floor(Int,x))
```

Comment:
Should use `T` instead of `Int`? Otherwise a problem if using a `Int64` distribution on a `Int == Int32` system.
```suggestion
    cdf(d, floor(T, x))
```